checkov/terraform/checks/graph_checks/aws/ALBWebACLConfiguredWIthLog4jVulnerability.yaml

```diff
@@ -0,0 +1,58 @@
+metadata:
+  id: "CKV_AWS_387"
```

Comment:
shouldn't it be `CKV2` if it's a graph check? (i.e - has a `cond_type: connection`)/
please validate across policies

Reply:
Good callout. I'll update those.